Tasks/Denkevich/Task8/ui.py

```diff
@@ -0,0 +1,124 @@
+import bl
+
+
+def show_all_categories():
+    return print(bl.get_all_categories())
+
+
+def show_goods_from_category(name):
+    return bl.get_goods_from_category(name)
+
+
+def refresh_the_cart(name, n):
+    return bl.minus_in_cart(name, n)
+
+
+def show_cart():
+    return bl.get_cart()
+
+
+def show_serials():
+    print(f"{bl.compact_serials()}\n")
+
+
+def order_finish():
+    bl.balance_reducing()
+    bl.prepare_cart_and_sum()
+    return print("Order is successfully finished.")
```

Comment:
never return print, just call it instead and return something else (or nothing at all)